releasenotes/notes/delwmidep-1219234765112adf.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    The Windows agent no longer depends on the Windows WMI service.  
+    If the WMI service stops for any reason, the Windows agent will no 
+    longer stop with it.  However, any integrations that do use WMI
+    will not be able to function until the WMI service restarts.
```

Comment:
Nice note 👍 

AFAIK at the moment 2 official integrations rely on WMI, `wmi_check` and `win32_event_log`, could you mention this here as well?